### DISCUSSION TITLE
Install package is downloaded really slow

### DISCUSSION THREAD

**Discussion by kecsolecso** [mockoon/mockoon#411] [Help] [answered]
3 minutes for windows installer
4 minutes for linux installer

I would say for a package size of 50 MB it is 2010 and not 2021

**Comment by 255kb** [comment, marked as answer]
We are storing binaries as GitHub Releases and have no plan to change this in the future. While it may not be the fastest hosting provider, we usually have decent perfs. I just downloaded the latest release in 5 seconds (from western Europe, using a smartphone sharing its 4G connection). 
You may want to check your network/setup.
If this is a recurring pattern among our users, we will surely investigate and look for another solution.
